backend/ContextAlgorithm/ContextAlgorithm.py

```diff
@@ -0,0 +1,133 @@
+import requests
+import xml.etree.cElementTree as ET
+import string
+
+INPUT_LANG = 'es'
+OUTPUT_LANG = 'en'
+WORD = 'cama'
+WORD_DIFFICULTY = 882
```

Comment:
Ok if that will change in the future its fine, else put a comment explaining that